File: app/src/main/java/com/moorhenapps/bluehex/utils/IntPoint.java
package com.moorhenapps.bluehex.utils;

public class IntPoint {
    public int x;
    public int y;

    public IntPoint(int x, int y) {
        this.x = x;
        this.y = y;
    }

    public IntPoint set(int x, int y) {
        this.x = x;
        this.y = y;
        return this;
    }
}

File: app/src/main/java/com/moorhenapps/bluehex/utils/IntPoint.kt
package com.moorhenapps.bluehex.utils

data class IntPoint(var x: Int, var y: Int) {

    fun set(x: Int, y: Int): IntPoint {
        this.x = x
        this.y = y
        return this
    }

    override fun toString(): String {
        return "$x,$y"
    }
}
